src/main/java/io/jenkins/plugins/coverage/CoveragePublisher.java

```diff
@@ -183,15 +180,6 @@ public void setTag(String tag) {
         this.tag = tag;
     }
 
-    public boolean getCalculateDiffForChangeRequests() {
-        return calculateDiffForChangeRequests;
-    }
-
-    @DataBoundSetter
-    public void setCalculateDiffForChangeRequests(boolean calculateDiffForChangeRequests) {
```

Comment:
Better mark as `@deprecated`, otherwise Pipelines using this flag will get errors